### PR TITLE
Update magic page

### PR DESCRIPTION
**Pull request by isoworg**

### CODE REVIEW

**Comment by isoworg** [comment]
@nikeshashar SQUIRREL ME PLEASE HUGS

File: data/magic_events.yml [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
+- date: Wednesday 5 April
+  event: Draft - AER/KLD or Cube
+
+- date: SUNDAY 23 April
   event: Pre-release - Amonkhet
 
-- date: Saturday 8 July 2017
-  event: Pre-release - Hour of Devastation
+- date: 8/9 July
+  event: CANCELLED Pre-release - Hour of Devastation
```
Comment (nikeshashar) [inline data/magic_events.yml]: If it's cancelled  do you still want it on the pagE?

Comment (isoworg) [inline data/magic_events.yml]: Yes, because people expect it to happen. I need to tell people that it's not happening.

**Comment by nikeshashar** [comment]
Sending it to our staging server.

**Comment by nikeshashar** [comment]
image

- [ ] Sunday in caps - inconsistent
- [ ] Strike through the cancelled event to make it clearer

**Comment by isoworg** [comment]
Sunday is in caps to draw attention to it being unusual.

**Comment by nikeshashar** [comment]
image


@isoworg how's this?

**Comment by isoworg** [comment]
👍

**Comment by nikeshashar** [comment]
We'll push this live on Monday @isoworg as we cannot deploy on Fridays

**Comment by nikeshashar** [comment]
:shipit:

**Pull request merged**